src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import core.basesyntax.io.read.ReportReader;
+import core.basesyntax.io.read.impl.ReportReaderImpl;
+import core.basesyntax.io.read.impl.csv.CsvReaderImpl;
+import core.basesyntax.io.write.ReportWriter;
+import core.basesyntax.io.write.impl.ReportWriterImpl;
+import core.basesyntax.io.write.impl.csv.CsvWriterImpl;
+import core.basesyntax.storage.impl.StorageImpl;
+import core.basesyntax.utils.convert.ReportConverter;
+import core.basesyntax.utils.convert.ReportConverterImpl;
+import core.basesyntax.utils.convert.csv.CsvConverterImpl;
+import core.basesyntax.utils.generate.ReportGenerator;
+import core.basesyntax.utils.generate.impl.ReportGeneratorImpl;
+import core.basesyntax.utils.generate.impl.csv.CsvGeneratorImpl;
+import core.basesyntax.utils.process.impl.ShopServiceImpl;
+import core.basesyntax.utils.transaction.FruitTransaction;
+import core.basesyntax.utils.transaction.handlers.OperationHandlerFactory;
+import java.io.FileNotFoundException;
+import java.util.List;
+
+public class Main {
+    private static final String CSV_FILE_PATH = "/";
+
+    public static void main(String[] args) {
+        ReportReader csvReader = new CsvReaderImpl(CSV_FILE_PATH);
```

Comment:
Direct instantiation of `CsvReaderImpl` with a hardcoded path goes against the checklist item to avoid hardcoding in your solution. Consider passing the file path as an argument to the `main` method or using a configuration file.